h2o-py/h2o/mli/_mli.py

```diff
@@ -0,0 +1,1738 @@
+        shap_summary="SHAP summary plot shows contribution of features for each instance. The sum "
+                     "of the feature contributions and the bias term is equal to the raw prediction "
+                     "of the model, i.e., prediction before applying inverse link function.",
+        pdp="Partial dependence plot (PDP) gives a graphical depiction of the marginal effect of a variable "
+            "on the response. The effect of a variable is measured in change in the mean response. "
+            "PDP assumes independence between the feature for which is the PDP computed and the rest.",
+        ice="Individual conditional expectations (ICE) plot gives a graphical depiction of the marginal "
+            "effect of a variable on the response. ICE plot is similar to partial dependence plot (PDP), "
+            "PDP shows the average effect of a feature while ICE plot shows the effect for a single "
+            "instance. The following plot shows the effect for each decile. "
+            "In contrast to partial dependence plot, ICE plot can provide more insight especially when "
+            "there is stronger feature interaction.",
+        ice_row="Individual conditional expectations (ICE) plot gives a graphical depiction of the marginal "
+                "effect of a variable on the response for a given row. ICE plot is similar to partial "
+                "dependence plot (PDP), PDP shows the average effect of a feature while ICE plot shows "
+                "the effect for a single instance.",
+        shap_explanation="SHAP explanation shows contribution of features for a given instance. The sum "
+                         "of the feature contributions and the bias term is equal to the raw prediction "
+                         "of the model, i.e., prediction before applying inverse link function. H2O implements "
+                         "TreeSHAP which when the features are correlated, can increase contribution of a feature "
+                         "that had no influence on the prediction.",
+    )
+
+    def __init__(self, for_what):
+        self.content = self.DESCRIPTIONS[for_what]
+
+    def _repr_html_(self):
+        return "<blockquote>{}</blockquote>".format(self.content)
+
+    def _repr_markdown_(self):
+        return "\n> {}".format(self.content)
+
+    def __repr__(self):
+        return self._repr_markdown_()
+
+
+@contextmanager
+def no_progress():
+    """
+    A context manager that temporarily blocks showing the H2O's progress bar.
+    Used when a multiple models are evaluated.
+    """
+    progress = h2o.job.H2OJob.__PROGRESS_BAR__
+    if progress:
+        h2o.no_progress()
+    yield
+    if progress:
+        h2o.show_progress()
+
+
+class NumpyFrame:
+    """
+    Simple class that very vaguely emulates Pandas DataFrame.
+    Main purpose is to keep parsing from the List of Lists format to numpy.
+    This class is meant to be used just in the MLI context.
```

Comment:
I would refrain from using the term, "MLI", since many people don't know that and we are focused on "explainability" right now.  I wonder if we should rename these modules?  `./explain/explain.py`?